### PR TITLE
Restructure tools and bindings

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
Ensures compatibility of tests with changes introduced in https://github.com/precice/precice/pull/630.

# Todo

- [x] take care of f2003 solverdummy and bindings

### CODE REVIEW

File: tests/Test_bindings.Ubuntu1804/Dockerfile [hunk 1]
```diff
@@ -29,25 +29,31 @@ RUN pip3 install --user .
...
+RUN git clone --branch $branch https://github.com/precice/fortran-module.git
+# Builds the fortran module
+WORKDIR /home/precice/fortran-module
+RUN make
 # Builds the f2003 solverdummy
-WORKDIR $PRECICE_ROOT/tools/solverdummies/f2003
-RUN gfortran $PRECICE_ROOT/src/precice/bindings/f2003/SolverInterfaceF2003.f90 -o solverdummy solverdummy.f03 $(pkg-config --libs-only-L --libs-only-l libprecice)
-RUN ./solverdummy ../precice-config.xml SolverOne MeshOne & ./solverdummy ../precice-config.xml SolverTwo MeshTwo
+WORKDIR /home/precice/fortran-module/examples/solverdummy
+RUN gfortran solverdummy.f03 -I../.. -L$(pkg-config --libs libprecice)
```
Comment (MakisH) [inline tests/Test_bindings.Ubuntu1804/Dockerfile]: Why not also use `make` here?

Comment (MakisH) [inline tests/Test_bindings.Ubuntu1804/Dockerfile]:
```suggestion
RUN make
```

File: tests/Test_bindings/Dockerfile [hunk 2]
```diff
@@ -34,25 +34,31 @@ RUN pip3 install --user .
...
+RUN git clone --branch $branch https://github.com/precice/fortran-module.git
+# Builds the fortran module
+WORKDIR /home/precice/fortran-module
+RUN make
 # Builds the f2003 solverdummy
-WORKDIR $PRECICE_ROOT/tools/solverdummies/f2003
-RUN gfortran $PRECICE_ROOT/src/precice/bindings/f2003/SolverInterfaceF2003.f90 -o solverdummy solverdummy.f03 $(pkg-config --libs-only-L --libs-only-l libprecice)
-RUN ./solverdummy ../precice-config.xml SolverOne MeshOne & ./solverdummy ../precice-config.xml SolverTwo MeshTwo
+WORKDIR /home/precice/fortran-module/examples/solverdummy
+RUN gfortran solverdummy.f03 -I../.. -L$(pkg-config --libs libprecice)
```
Comment (MakisH) [inline tests/Test_bindings/Dockerfile]: Same here
```suggestion
RUN make
```

**Review by MakisH — APPROVED** [review]
I did not run it, but the code looks good! I would just change the explicit compilation command to just `make`. This is more future-proof.

File: tests/Test_bindings.Ubuntu1804/Dockerfile (hunk 1 above)
Comment (BenjaminRodenberg) [reply tests/Test_bindings.Ubuntu1804/Dockerfile]: totally oversaw the Makefile :)

**Pull request merged**